cmd/titus-executor-backend/main.go

```diff
@@ -87,6 +87,11 @@ func main() {
 	defer m.Flush()
 
 	app.Action = func(c *cli.Context) error {
+		if mainCfg.debug {
```

Comment:
Huh. I thought for sure debug logs were enabled in the *standalone* tests, is this a different class of debugs or a different class of integration tests that you are refering to?

Reply:
A bit of a naming error on my part - this is fixes for titus-functional-tests that run against real Titus stacks, not the integration tests in this repo.  Sorry for the confusion. :\